### PR TITLE
Update features for UG

### PR DESCRIPTION
**Pull request by donwong2308**

### CODE REVIEW

**Review by PallonCX — APPROVED** [review]
Perhaps we can add more detail for each commands, specifically what fields it can take and the use case with some screenshot.

**Review by larainezo — APPROVED** [review]
LGTM

**Review by JustWeiHao — COMMENTED** [review]
Other than that, LGTM

**Review by nyankoclaws — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -85,6 +85,10 @@ or to [Command Summary](#command-summary) for a summary of the commands.
 - Items in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
+- Free Time Tags are in the following format: `DDD:HH:mm` <br>
+- `DDD` is from Mon-Sun, `HH:mm` is 24 hour time format <br>
+  e.g. **Mon:1300-1400**
+
```
[the remark this answers is not in the document]
Comment (nyankoclaws) [reply docs/UserGuide.md:91]: Yea i think it should be...

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Pull request merged**